websites/A/AniWatch/metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "819696394180100106",
+		"name": ".vahya"
+	},
+	"service": "AniWatch",
+	"description": {
+		"en": "AniWatch.to - A site to watch anime online for Free"
+	},
+	"url": "aniwatch.to",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/LvFXasi.png",
+	"thumbnail": "https://i.imgur.com/TAc6lvS.png",
+	"color": "#ffdd95",
+	"category": "anime",
+	"tags": [
+		"anime"
+	],"iframe": true,
+	"iFrameRegExp": ".*",
```

Comment:
Can you use a better regex?

Reply:
I tried using "iframe-embed", which is the id of all iframes, but it doesn't work, and I can't inspect iFrames; the site reloads if you do an inspect.